db/batchlog_manager.cc

```diff
@@ -31,13 +31,9 @@
 #include "gms/failure_detector.hh"
 #include "gms/gossiper.hh"
 #include "schema_registry.hh"
-#include "idl/uuid.dist.hh"
 #include "idl/frozen_schema.dist.hh"
-#include "serializer_impl.hh"
-#include "serialization_visitors.hh"
-#include "db/schema_tables.hh"
-#include "idl/uuid.dist.impl.hh"
```

Comment:
Nice. I always hated the obscure error generated by these missing headers and how sensitive it all was to ordering.